tbx/static_src/sass/components/_footer-cta.scss

```diff
@@ -67,4 +67,18 @@
     &__button {
         padding: 12px 20px;
     }
+
+    &__email-wrapper {
+        margin-top: $spacer-small;
+        margin-bottom: $spacer-small;
+
+        @include media-query(large) {
+            margin-bottom: 0;
+        }
+    }
+
+    &__email {
+        @include link-styles();
+        //@include focus-styles();
```

Comment:
Looks like the focus styles are working without this so can probably be deleted : )

Reply:
whoopsy...